buildkite/run_check_deploy_docker_build.sh

```diff
@@ -4,20 +4,10 @@
 # built successfully. This check is done in the buildkite-assets repo to prevent
 # an unbuildable Dockerfile from being deployed to buildkite instances.
 
-DOCKER_IMAGE_NAME=check-heroku-deploy-image
+echo "--- Building and testing with panorama_command.sh"
+HEROKU_DEPLOY_PATH=deploy/heroku-deploy \
+  HEROKU_DEPLOY_IMAGE_NAME=check-heroku-deploy-image \
+  DEPLOYMENT_APP_NAME=panorama-addons \
```

Comment:
The built Docker container uses `heroku_deploy.sh` as its entrypoint, which [requires a valid Heroku app name in the `DEPLOYMENT_APP_NAME` env var](https://github.com/panorama-ed/buildkite-assets/blob/main/deploy/heroku-deploy/heroku_deploy.sh#L22). `panorama-addons` has no functionality, and we don't actually run a command, so this is just a safe value.